extdata/gis-vs-gds-table.csv

```diff
@@ -1,4 +1,4 @@
-Attribute,GIS,GDS
-"Home disciplines","Geography","Geography, Computing, Statistics"
-"Software focus","Graphical User Interface","Code"
+"Attribute","Desktop GIS (GUI)","R"
```

Comment:
Fantastic, you made the change! Could we not make it about Geocomputation though?

Geocomputation with R being one pathway to the geocomputational approach that we expound in the book.